policy/multipolicyloader.go

```diff
@@ -96,7 +109,12 @@ func (l *MultiPolicyLoader) Load() (*Policy, Source, error) {
 // full root privileges.
 func ReadWithSudoScript(h *HomePolicyLoader, username string) ([]byte, error) {
 	// opkssh readhome ensures the file is not a symlink and has the permissions/ownership.
-	cmd := exec.Command("sudo", "-n", "/usr/local/bin/opkssh", "readhome", username)
+	opkBin, err := os.Executable()
```

Comment:
```suggestion
	// The default path is /usr/local/bin/opkssh
	opkBin, err := os.Executable()
```

Add this comment so that if someone is doing a find on "/usr/local/bin/opkssh" they will find this code.

Reply:
Thanks for remove ammended